onnxruntime/core/util/qmath.cc

```diff
@@ -60,4 +68,33 @@ void QGemmu8u8_s32(
   GemmlowpMultiplyu8u8_s32(lhs_data, rhs_data, result_data, lhs_offset, rhs_offset, M, N, K, thread_pool);
 #endif
 }
+
+template void QGemm<uint8_t, int8_t, int32_t>(
```

Comment:
What error were you getting without these templates here?

Reply:
no need to explicit instantiation here.